iocore/net/UnixNetAccept.cc

```diff
@@ -221,11 +247,26 @@ NetAccept::init_accept_per_thread()
   }
 
   period = -HRTIME_MSECONDS(net_accept_period);
-  n      = eventProcessor.thread_group[opt.etype]._count;
+  REC_ReadConfigInteger(accept_threads, "proxy.config.accept_threads");
```

Comment:
Just realized that we're reading this config quite a few times throughout the code, 5 times if including this one. I think reading it once to a global variable might be better.